### PR TITLE
Settings as environment variables

### PR DESCRIPTION
**Pull request by mkurek**
- renamed db and redis settings names to be more generic
- all (important) settings could be now set by environment variables
- improved logging config

### CODE REVIEW

**Comment by szok** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

File: src/ralph/settings/prod.py [hunk 1]
```diff
@@ -2,12 +2,10 @@
...
 
-STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'  # noqa
-STATIC_ROOT = os.path.join(BASE_DIR, 'var', 'static')
-
-# FIXME: when going for full production, change it to False
+DEBUG = False
+TEMPLATE_DEBUG = False
 
-DEBUG = True
+STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'  # noqa
```
Comment (vi4m) [inline src/ralph/settings/prod.py:9]: Does it need collecstatic now to serve local files?

**Comment by andrzej-jankowski** [comment]
:ok:

**Comment by mkurek** [comment]
I've set `DEBUG` to `True` in prod settings - we'll back to this topic in a while.

**Comment by vi4m** [comment]
Review for review: There were 3 revievers involved, and nobody discovered that the documentation installation (http://ralph-ng.readthedocs.org/installation/#settings) was not updated.

**Pull request merged**